assets/javascript/script.js

```diff
@@ -163,29 +157,29 @@ function wJax(googleOutput) {
   });
 }
 
-function weatherAjax(input) {
+function weatherAjax(address) {
```

Comment:
changed variable so it's understandable, and changed formatting of code so it's readable